runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultIllumina.java

```diff
@@ -173,20 +179,28 @@ public NotificationDto process(File runDirectory, TimeZone tz) throws IOExceptio
     // The Illumina library can't distinguish between a failed run and one that either finished or is still going. Scan the logs, if
     // available to determine if the run failed.
     File rtaLogDir = new File(runDirectory, "/Data/RTALogs");
-    boolean failed = Optional
-        .ofNullable(rtaLogDir.listFiles(file -> file.getName().endsWith("Log.txt") || file.getName().endsWith("Log_00.txt")))
-        .map(files -> Arrays.stream(files)
-            .anyMatch(file -> {
-              try (Scanner scanner = new Scanner(file)) {
-                return scanner.findWithinHorizon(FAILED_MESSAGE, 0) != null;
-              } catch (FileNotFoundException e) {
-                log.error("RTA file vanished before reading", e);
-                return false;
-              }
-            }))
-        .orElse(false);
-    if (failed) {
+    LocalDateTime failedDate = Arrays
+        .stream(rtaLogDir.listFiles(file -> file.getName().endsWith("Log.txt") || file.getName().endsWith("Log_00.txt")))
+        .map(file -> {
+          try (Scanner scanner = new Scanner(file)) {
+            String failMessage = scanner.findWithinHorizon(FAILED_MESSAGE, 0);
+            if (failMessage == null) {
+              return null;
+            }
+            Matcher m = FAILED_MESSAGE.matcher(failMessage);
+            return LocalDateTime.parse(m.group(1), FAILED_MESSAGE_DATE_FORMATTER);
+          } catch (FileNotFoundException e) {
+            log.error("RTA file vanished before reading", e);
+            return null;
+          }
+        })
+        .filter(Objects::nonNull)
+        .findAny()
```

Comment:
I think you should `.sorted(LocalDateTime::compareTo).findFirst()` to get the earliest date.